### PR TITLE
Cleanup transfer.md page, remove CRI-O content

### PR DESCRIPTION
**Pull request by rhatdan**
This page has not been updated since we copied it from CRI-O repo.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: transfer.md [hunk 1]
```diff
@@ -20,9 +18,9 @@ As well as some systemd helpers like `systemd-cgls` and `systemd-cgtop` are stil
 ## Equivalents
 
 For many troubleshooting and information collection steps, there may be an existing pattern.
-Following provides equivalent with PODMAN tools for gathering information or jumping into containers, for operational use.
+Following provides equivalent with `podman tools for gathering information or jumping into containers, for operational use.
```
Comment (mheon) [inline transfer.md]: Missing a backtick after podman

File: transfer.md [hunk 2]
```diff
@@ -1,16 +1,14 @@
...
 ## Abstract
 
-The `crio` daemon is intended to provide the [CRI](https://github.com/kubernetes/community/blob/master/contributors/devel/container-runtime-interface.md) socket needed for Kubernetes to use for automating deployment, scaling, and management of containerized applications (See the document for [configuring kubernetes to use PODMAN](./kubernetes.md) for more information on that).
-Therefore the `crioctl` command line is a client that interfaces to the same grpc socket as the kubernetes daemon would, for talking to the `crio` daemon.
-In many ways `crioctl` is only as feature rich as the Kubernetes CRI requires.
-There are additional tools e.g. `podman` and [`buildah`](https://github.com/projectatomic/buildah) that provide a feature rich set of commands for all operational needs in a Kubernetes environment.
-
+podman is a tool for managing Pods, Containers, and Container Images.  The CLI
+for podman is based on the Docker CLI, although podman does not require a
+container daemon to be running in order to function.
```
Comment (mheon) [inline transfer.md]: Maybe runtime daemon instead of container daemon?

Comment (TomSweeneyRedHat) [reply transfer.md]: I'd pref "Podman" through out with the exception of commands.

File: transfer.md (hunk 1 above)
Comment (rhatdan) [reply transfer.md]: Fixed

File: transfer.md (hunk 2 above)
Comment (rhatdan) [reply transfer.md]: Fixed

**Comment by mheon** [comment]
LGTM
@rh-atomic-bot r+

**Pull request closed without merging**